charts/prefect-server/values.yaml

```diff
@@ -340,8 +342,6 @@ postgresql:
     persistence:
       # -- enable PostgreSQL Primary data persistence using PVC
       enabled: false
-      # -- PVC Storage Request for PostgreSQL volume
-      size: 8Gi
```

Comment:
does this default need to be re-implemented somewhere?

Reply:
Shouldn't need to, since it's [the default in Bitnami's values.yaml](https://github.com/bitnami/charts/blob/postgresql/12.12.10/bitnami/postgresql/values.yaml#L685). We could technically leave this here, but I deleted it to make it more clear that everything defined under `postgresql` is an override for the chart's defaults.